components/widgets/menu/menu_header.tsx

```diff
@@ -8,14 +8,18 @@ import './menu_header.scss';
 type Props = {
     divider?: React.ReactNode;
     children?: React.ReactNode;
+    onClick?: () => void;
 }
 
 export default class MenuHeader extends React.PureComponent<Props> {
     public render() {
-        const {children} = this.props;
+        const {children, onClick} = this.props;
```

Comment:
Architecturally speaking this feels a touch wonky, since to me this is purely a wrapper component. Ideally I think we could toss an anchor around the contents of the header and that would be our clickable target.

1/5 though, just a suggestion.

Reply:
Initially i did try this and in fact it caused a lot of styling issues, new menu incoming though :)